components/suggest/src/db.rs

```diff
@@ -140,17 +140,17 @@ impl<'a> SuggestDao<'a> {
         {
             (self.conn.prepare_cached(
                 &format!(
-                    "SELECT s.id, k.rank, s.title, s.url, s.provider, NULL as confidence, NULL as keyword_suffix
+                    "SELECT s.id, k.rank, s.title, s.url, s.provider, IFNULL(s.score, 0.2) AS score, NULL as confidence, NULL as keyword_suffix
```

Comment:
I'd ❤️ to see this in a constant eventually—like `DEFAULT_SUGGESTION_SCORE` to match Desktop's naming—so that we aren't embedding magic numbers in our queries, but that can be in a follow-up.

Reply:
Oh, I was thinking that we leave the database field as nullable, so that if we ever wanted to experiment with changing the default score, we'd have a way of distinguishing them!

Reply:
> Oh, I was thinking that we leave the database field as nullable, so that if we ever wanted to experiment with changing the default score, we'd have a way of distinguishing them!

I think using a default score in the `suggestion` table should be fine. For experimentation, we can always override it via external sources (like what we do for AMP & Pocket suggestions). In the longer term, I'd foresee a path that we'd adjust the `score`s from DB for relevance ranking perhaps in the application.

Reply:
Sounds good to me, thanks! 👍🏼